docker-entrypoint.sh

```diff
@@ -14,6 +14,13 @@ echo "alias devto-start='cd /usr/src/app/ && bundle exec rails server -b 0.0.0.0
 #
 cd /usr/src/app/
 
+#
+# Clean out server.pid
```

Comment:
Technically not necessary for this PR, but useful when someone runs a Docker build while having a server running.